mandatory/2-function-creation.js

```diff
@@ -16,7 +19,15 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if(num>100){
+    return "100%"
+  }else if(num%1!==0){
+    return Math.round((num + Number.EPSILON) * 100) / 100 +"%"
```

Comment:
Not sure what this line of code does. But if you are trying to round a number to 2 decimal places, try this instead:

`num.toFixed(2)`

If you want to remove trailing zeroes, this will work:

`+num.toFixed(2)`